client/src/components/Jobs/Jobs.jsx

```diff
@@ -112,6 +112,7 @@ function JobsView({ showDetails, showFancyStatus, rowProps }) {
       headerStyle: { textAlign: 'left' },
       accessor: '_links.archiveData.href',
       Cell: (el) => {
+        // TODO V3: Handle links differently (also in jobsUtil)
         const outputPath =
           el.row.original.outputLocation || getOutputPathFromHref(el.value);
```

Comment:
We should be able to grab the output path straight from the job listing now, if we can `select` the attributes we need from the response.